### ISSUE TITLE
Request: enable/disable all breakpoints button on main window's toolbar

### ISSUE THREAD

**Issue by IgorChernyshov** [ProxymanApp/Proxyman#743]
Proxyman version 2.15.2
MacOS version 10.15.7

Feature request:
It would be great to enable/disable all breakpoints with a button on main window's toolbar. Currently I have to go to Breakpoints -> Add Rule -> Enable Breakpoint Tool and I do it quite often.

Thanks!

**Comment by NghiaTranUIT** [comment]
Hey, how about open the Breakpoint List by **⌥⌘B** and hit **⌘E** to enable/disable the entire Breakpoint Tool 🤔 

Is it useful than opening the Breakpoint from the menu?

I'm thinking that what if I introduce other shortcuts (by appending ⇧ key) to enable/disable a particular tool, such as:
- ⌥⌘B: Open Breakpoint List => ⇧⌥⌘B to enable/disable from the main window
- ⌥⌘I: Open Scripting List => ⇧⌥⌘I to
- ⌥⌘L: Open Map Local List => ⇧⌥⌘L
- and so all

**Comment by IgorChernyshov** [comment]
Hi!

These are great news, thanks!

Igor Chernyshov Sent from iPhone

Friday, 27 August 2021, 18:06 +0300 from ***@***.***  ***@***.***>:

**Comment by tibioneh** [comment]
This is a great tool but it really needs a button somewhere (top-left side would work) and a keyboard shortcut for just Enabling/Disabling the Breakpoint tool. It's just a pain if you use it often otherwise. 
Please try to add this sometime soon, thank you.

**Comment by NghiaTranUIT** [comment]
Hey @tibioneh, you can toggle the Breakpoint from the Menu Bar.

<img width="448" alt="Screen Shot 2022-08-26 at 16 16 16" src="https://user-images.githubusercontent.com/5878421/186871328-0febef3b-0f0c-46e7-9a5d-6c70ec229ba7.png">

**Comment by tibioneh** [comment]
Thank you, I found out that you can customize the toolbar and add a button there shortly after I posted the comment, that's why I deleted it :).
I knew about your suggestion but I was hoping for a keyboard shortcut - it would be quite useful as you often keep switching between the 2 modes when testing.

**Comment by tibioneh** [comment]
One more question if you don't mind: is it possible to change a response body automatically?

**Comment by NghiaTranUIT** [comment]
@tibioneh yes, you can either use Map Local or Scripting to automatically change the body.

## Map Local: 
- How to use: Right-Click on the Request -> Tools -> Map Local -> You can change the Status Code, Header, Body of the Response.
- Document: https://docs.proxyman.io/advanced-features/map-local

## Scripting
- How to use: Right-Click on the Request -> Tools -> Scripting -> Create a rule -> Open the Script -> Use the following snippet code to modify the Body, Header, of both Request / Response
- Snippet Code: https://docs.proxyman.io/scripting/snippet-code

**Comment by tibioneh** [comment]
Great, thanks a lot!
